src/main/java/com/jadventure/game/prompts/CommandCollection.java

```diff
@@ -84,8 +80,19 @@ public void command_help() {
                 continue;
             }
             Command annotation = method.getAnnotation(Command.class);
-            String command = (annotation.aliases().length() == 0) ? 
-                annotation.command() : annotation.command() + " (" + annotation.aliases() + "):"; 
+            StringBuilder command;
+            if (Arrays.equals(annotation.aliases(), new String[]{""})) {
+                command = new StringBuilder(annotation.command());
+            } else {
+                command = new StringBuilder(annotation.command() + " (");
+                for (int i = 0; i < annotation.aliases().length; i++) {
+                    if (i == annotation.aliases().length - 1)
+                        command.append(annotation.aliases()[i]);
+                    else
+                        command.append(annotation.aliases()[i]).append(", ");
+                }
+                command.append("):");
+            }
```

Comment:
Part of this happens in either case, so rather
```suggestion
           command = new StringBuilder(annotation.command());
           if (annotation.aliases().length > 0) {
                command.append(" (");
                for (int i = 0; i < annotation.aliases().length; i++) {
                    if (i == annotation.aliases().length - 1)
                        command.append(annotation.aliases()[i]);
                    else
                        command.append(annotation.aliases()[i]).append(", ");
                }
                command.append("):");
            }
```